docs/pages/components/toggle.md

```diff
@@ -14,189 +14,171 @@ The toggle/switch is meant to resemble a physical switch and allow a user to tur
 It is recommended to always display the toggle with a label above it as well as the label of the selected state. For example, the label above would be Active, the toggle state would be “on” and the selected state label displayed to the right of the toggle would be “Yes”.
 
 
-## Toggle with Label on top
-To clarify the meaning of the toggle, use a label above it (ex. Live).
+## Toggle with Labels on Sides
+This is basic usage of toggle
 
 {% capture default %}
 <div class="fd-form-group">
     <div class="fd-form-item">
-        <label class="fd-form-label" for="y21YO391">Live:</label>
+        <label class="fd-form-label">Toggle with label</label>
         <label class="fd-form-label fd-form-label--toggle">
             <span class="fd-toggle">
-                <input class="fd-toggle__input" type="checkbox" name="" value="" id="y21YO391">
-                <span class="fd-toggle__switch" role="presentation"></span>
+                <input class="fd-toggle__input" type="checkbox" name="" value="" id="y21YO3911">
+                <div class="fd-toggle__wrapper">
+                    <div class="fd-toggle__inner">
+                        <span class="fd-toggle__label fd-toggle__label--on">on</span>
```

Comment:
`fd-switch__text`

Reply:
I wanted to use `fd-toggle__label` not only for text, it's also for icons.

Reply:
I will use `__text`, because `__label` is reserved for labels